llvm/test/Instrumentation/AddressSanitizer/AMDGPU/asan_instrument_mem_intrinsics.ll

```diff
@@ -0,0 +1,67 @@
+;RUN: opt < %s -passes=asan -S | FileCheck %s
```

Comment:
Use -mtriple instead of target triple. Also use generated checks instead of manually written, almost generated checks